### ISSUE TITLE
[HTML5] Coalesce operator and $-accessor do not work correctly

### ISSUE THREAD

**Issue by Grisgram** [YoYoGames/GameMaker-Bugs#5088]
### Description
Hey there,

attached project demonstrates the bug.

Creating a temp structure with 2 functions.
Those will be accessed through the $ accessor.

 ```
test = {
	fallback_func: function(a) { show_debug_message($"--- FALLBACK: {a}"); },
	regular_func: function(a) { show_debug_message($"--- REGULAR: {a}"); },
}
```

Then, the coalesce operator is used to determine, whether "regular_func" exists, and shall return "fallback_func", if not.
	
* In Windows, all is good
* In HTML, the "fallback" line is silently skipped

```
test[$ "fallback_func"]("Test of fallback: OK");
test[$ "regular_func"]("Test of regular: OK");

show_debug_message("-- before accessing fallback through coalesce");

// THIS LINE BELOW DOES NOT WORK IN HTML
(test[$ "does_not_exist"] ?? test[$ "fallback_func"])("fallback is hit!");

show_debug_message("-- after accessing fallback through coalesce");
```

Run this on windows target, and you get this log output:
```
--- FALLBACK: Test of fallback: OK
--- REGULAR: Test of regular: OK
-- before accessing fallback through coalesce
--- FALLBACK: fallback is hit!
-- after accessing fallback through coalesce
```

run it on html and you see, the ?? line is silently skipped (log taken from "Console" of chrome when starting in HTML):
```
--- FALLBACK: Test of fallback: OK
--- REGULAR: Test of regular: OK
-- before accessing fallback through coalesce
-- after accessing fallback through coalesce
```

My guess is, that the html-version of `test[$ "does_not_exist"]` does not return a nullish value if the element does not exist, and therefore the `??` coalesce is not triggered.

### Steps To Reproduce
* Just run the attached project

### Which version of GameMaker are you reporting this issue for?
IDE v2024.2.0.132 Runtime v2024.2.0.163

### Which operating system(s) are you seeing the problem on?
Windows 10.0.19045.0

[d27e5161-88fa-4033-a159-0a03ec932774](https://api.gamemaker.io/api/github/downloads/d27e5161-88fa-4033-a159-0a03ec932774)

**Comment by rwkay** [comment]
Fixed in 2024.6

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.600.0.561  Runtime v2024.600.0.579
